src/main/java/org/folio/consortia/domain/dto/Permission.java

```diff
@@ -1,4 +1,12 @@
 package org.folio.consortia.domain.dto;
 
-public record Permission(String permissionName) {
+import lombok.AllArgsConstructor;
```

Comment:
If constructor is used instead of setter this change won't be needed

Reply:
Changed to record